apps/pie-microsite/test/helpers/configuration-helper.js

```diff
@@ -3,8 +3,8 @@
 // This function is used to set the correct URL for WebDriverIO tests.
 //
 // local - http://localhost:8080/
-// 'main' branch - Github Page - https://pie.design/
-// Pull Request - GitHub Pages Preview URL - https://pie.design/pr-preview/pr-<pr-number>
+// 'main' branch - Github Page - https://pie.design/docs
```

Comment:
Is the URL actually changing for the site, or is this just for testing?

Reply:
I believe the plan is for it to change for now. We want to look into the CNAME subdomain stuff eventually that we talked about this morning though. Nothing should be referencing this site yet so it shouldn't pose any risk ahead of March

This way we'd have:

pie.design/docs
pie.design/storybook

Reply:
Unfortunately, yes :(

This is the limitation with deploying multiple apps to GitHub Pages. Essentially each app is a directory in the `gh-pages` branch, and the routing is based off folder structure.

We can probably address this with a new `CNAME` for each app, and use subdomains like `docs.pie.design` / storybook.pie.design`

A more scalable approach though, would be to use a deployment mechanism that's fit-for-purpose, such as Netlify / S3 / Vercel, but obviously this involves 💰 💰 💰

Reply:
I would probably make sure we add a `robots.txt` to the base of the website for now then if we're playing around with URLs, otherwise we're going to end up potentially getting punished by Google in terms of SEO ranking for broken links on our domain.

If we're making this change, does going to `pie.design` redirect to `pie.design/docs`, or will it break after this PR. I'd probably try and get a basic redirect in so that we don't end up with broken links in search history if possible.

Reply:
Out of interest, why do we need to move the docs site out of the root? Couldn't storybook just be deployed to a subfolder, or is it more complex than that?

Reply:
> Out of interest, why do we need to move the docs site out of the root? Couldn't storybook just be deployed to a subfolder, or is it more complex than that?

Will look into this

Reply:
I believe this is now sorted. We'll be using the following URL's.


Main Microsite - pie.design
Main Storybook - pie.design/storybook

PR Microsite  - pie.design/pr-preview-docs/{pr_number}
PR Storybook  - pie.design/pr-preview-storybook/{pr_number}